### DISCUSSION TITLE
Volunteer myself for update documentation on how to solved problem with global object due to outdated version of react-floater

### DISCUSSION THREAD

**Discussion by CosmicTiger** [gilbarbara/react-joyride#900] [Ideas]
Hello, @gilbarbara

I'm using recently your library and I have been really interested in try to provide you some help if you will and like on how to solved the issue related over the global object mentioned in this issue [Do you plan to update the version to react-floater v0.8.x?](https://github.com/gilbarbara/react-joyride/discussions/820). Also, I believe could be a good idea if we could update the documentation on some regard to extend some usability case scenario for the library. This is a proposal for me to help you but also you could take it as general idea to have some docs on how to fix the issue more clearly for people while you still work on how to update the library. Even tho, let me know what are you thoughts 😄

**Comment by gilbarbara** [comment]
Hey @CosmicTiger 

The "problem" with the `global` variable isn't a bug with this library.
https://developer.mozilla.org/en-US/docs/Glossary/Global_object
https://nodejs.org/api/globals.html#globals_global

Vite chose not to provide a fallback, while webpack does it. 
If you check Vite's documentation for the [define option](https://vitejs.dev/config/shared-options.html#define), they explicitly mention it.
So a quick web search for "global is not defined" would show this page and how to fix it for that particular bundler.

We could add a FAQ to the docs with some common pitfalls. However, people often don't read the documentation, so this may not have an impact.
